gui-apps/foot/foot-1.22.3.ebuild

```diff
@@ -89,4 +89,7 @@ src_install() {
 
 pkg_postinst() {
 	xdg_pkg_postinst
+
+	optfeature "URI launching" x11-misc/xdg-utils
+	optfeature  "Desktop notifications" x11-libs/libnotify
```

Comment:
double space, also can you provide a more thorough explanation?

Reply:
I meant an elaboration in the actual optfeature string, so that users who see this when they freshly install `foot` can decide whether they actually want the features